test/torchaudio_unittest/transducer_test.py

```diff
@@ -271,6 +280,6 @@ def test_costs_and_gradients_B2_T4_U3_D3_fp32(self):
         self.assertEqual(gradients, ref_gradients, atol=atol, rtol=rtol)
 
 
-@common_utils.skipIfNoExtension
-class CPUTransducerTester(TransducerTester, common_utils.PytorchTestCase):
+@skipIfNoRNNT
```

Comment:
as above:
```suggestion
@skipIfNoTransducer
```

Reply:
Thanks a lot for the suggestion! its very rigorous I got to keep up with your standard. will change it first